shell.nix

```diff
@@ -3,7 +3,7 @@
 pkgs.mkShell {
   nativeBuildInputs = with pkgs; [
     (rust-bin.stable.latest.default.override { extensions = ["rust-src"]; })
-    lld_10
+    # lld_10
```

Comment:
What does this do?

Reply:
my understanding it's a toolchain but @archseer also has it disabled in a PR that's updating even more dependencies https://github.com/smartcontractkit/chainlink-solana/pull/588/files#diff-e53dfbfffe62ae3c0b411b3938ccffa9fb6a2ecc565f55785ef8daa756631a6b